lib/ts/types.ts

```diff
@@ -109,6 +109,11 @@ export type RecipeInterface = {
     doesSessionExist: (input: { userContext: any }) => Promise<boolean>;
 
     signOut: (input: { userContext: any }) => Promise<void>;
+
+    validateClaims: (input: {
+        claimValidators: SessionClaimValidator[];
+        userContext?: any;
+    }) => Promise<ClaimValidationError[] | undefined>;
```

Comment:
Having the return type be undefined is a bit weird? When the same thing can also be signified with an empty array being returned? Cause now, the user will have to do:
```
let errors = await validateClaims(...)

if (errors !==undefined && errors.length > 0) {
  ... handle error
}
```

As opposed to just doing `errors.length > 0` check

Reply:
@porcellus missed this?